File: refdiff-universal/src/test/resources/php/refactor/extract/v1/main.php
<?php

function extracted_method() {
  $x = 10;
  $y = 20;
  echo ($x + $y) . PHP_EOL;
}

function foo() {
  extracted_method();
  echo "hello from foo" . PHP_EOL;
}

function bar() {
  echo "hello from bar" . PHP_EOL;
}

File: refdiff-universal/src/test/resources/go/refactor/extract/v1/main.go
package main

import "fmt"

func extracted_method() {
	x := 10
	y := 20
	fmt.Println(x + y)
}

func foo() {
	extracted_method()
	fmt.Println("hello from foo")
}

func bar() {
	fmt.Println("hello from bar")
}
